integration-tests/api/__tests__/product-categories/materialized-path.ts

```diff
@@ -0,0 +1,74 @@
+import path from "path"
```

Comment:
thought(non-blocking, this-file): We could move this to a new folder specifically for repository level integration tests `root/integration-tests/repository`

Reply:
Will tackle this as a separate PR tomorrow, since that will muddy the PR a bit. I'll also have to move some other tests in there as well. @adrien2p - what you've been looking for ;)